ckanext/data_qld/templates/reporting/index.html

```diff
@@ -51,42 +51,42 @@ <h2>Organisation: {{ org_title }}</h2>
             <tbody>
                 <tr>
                     <td>{{ _('Dataset') }} {{ _('followers') }}</td>
```

Comment:
can we add an id for the titles also. ie. "${myname}_title"
or 
rejig to put an id on the tr, then simple class on the td's
tr id="dataset_followers"
td class="metric-title, metric-${type i.e. followers}"
td class="metric-data, metric-${type i.e. followers}"
then you can reference and check both fields are valid and in correct order too

Reply:
@duttonw I have updated as you requested and added the 'id' to the 'tr' rows and added classes to the metric columns.
The 'metric-title' & 'metric-data' values & column positions are now checked and verified in the 'reporting.feature'